machine_learning_hep/data/database_ml_parameters_LcpK0spp.yml

```diff
@@ -199,7 +199,7 @@ LcpK0spp:
     data:
       nperiods: 4
       nprocessesparallel: 20
-      maxfiles : [40,40,40,40] #list of periods
+      maxfiles : [20,20,20,20] #list of periods
```

Comment:
set to -1, please, so that we can run the full production with DBs as in the repo